src/ekat/kokkos/ekat_subview_utils.hpp

```diff
@@ -316,6 +314,177 @@ subview_1(const ViewLR<ST******,Props...>& v,
       v.impl_track(),vm);
 }
 
+// ================ Multi-sliced Subviews ======================= //
+// e.g., instead of a single-entry slice like v(:, 42, :), we slice over a range
+// of values, as in v(:, 27:42, :)
+// this means that the subview has the same rank as the source view
```

Comment:
I would add a comment starting that the pair of indices is to be interpreted as `[beg,end)`, so the second index is the first _after_ the desired ones (just like the end iterator in std containers).